.github/workflows/test-dbt-installation-pip.yml

```diff
@@ -43,23 +43,8 @@ defaults:
     shell: bash
 
 jobs:
-  generate-artifact-name:
-    runs-on: ubuntu-latest
-
-    outputs:
-      artifact-name: ${{ steps.artifact-name.outputs.name }}
-
-    steps:
-      - name: "Generate Artifact Name"
-        id: artifact-name
-        run: |
-          date=$(date +'%m%d%Y-%H%M%S')
```

Comment:
would it be easier to have artifact-naming as a reusable action that all our workflows could use?

Reply:
We're removing the artifacts from this process entirely as they were only used to trigger the notifications. So I'll assume this is a general question outside of the scope of this PR.

A reusable action for naming artifacts is one way in which we could enforce some naming standards. For example, we always append the date in a certain way, everything is underscore-separated, etc. And it would also allow for us to provide a "composite key" of sorts. At the same time, it feels like this action would essentially be a single step. There might be a step before to debug-log inputs, and a step after for debug-logging the result. I don't know if the benefit necessarily outweighs the cost; both are minor in my opinion.

Reply:
GitHub artifacts are short lived.  Not sure it's worth it.